tests/unit/memefficiency.tcl

```diff
@@ -298,6 +298,14 @@ run_solo {defrag} {
                 $rd read ; # Discard replies
             }
 
+            set rd1 [redis_deferring_client]
+            if {$type eq "standalone"} {
+                $rd1 subscribe chan
+            } else {
+                $rd1 ssubscribe chan
+            }
+            $rd1 read
```

Comment:
any reason not to subscribe to both (regardless of the `$type`)? i.e. the API is supported either way.
we can use the same client, right?

p.s. let's add a comment that says this is here for code flow coverage.